### ISSUE TITLE
'popover.Constructor' is undefined error occurs immediately as I run my meteor project

### ISSUE THREAD

**Issue by KaleenAli** [nate-strauser/meteor-x-editable-bootstrap#40]
So I recently downloaded this package into my meteor project, and right away when I started my app it threw this error.

<img width="766" alt="screen shot 2015-12-16 at 10 47 40 am" src="https://cloud.githubusercontent.com/assets/10760228/11847527/91e8ed10-a3e3-11e5-9833-fe9e43d02dce.png">

bootstrap-editable.js line 4707 is below: 
<img width="383" alt="screen shot 2015-12-16 at 10 47 54 am" src="https://cloud.githubusercontent.com/assets/10760228/11847755/8f3deaba-a3e4-11e5-94b4-9c93260c0cac.png">

When researching the problem I have found some possible causes, but have not been able to solve the error.  

One theory is that popover.js is being loaded after x-editable.js - if this is true, does anyone know how I can create a specific order?

Also, I personally am not looking to use the Default Popover form, I'm looking to implement the inline version of the package.  If there is an easy way to just ignore the popover code, that'd work for me as well!

Any help will be greatly appreciated! 
Here is the list of packages within my project:
<img width="622" alt="screen shot 2015-12-16 at 10 12 47 am" src="https://cloud.githubusercontent.com/assets/10760228/11847892/2f8eb2ce-a3e5-11e5-95b5-de19943c259d.png">

**Comment by nate-strauser** [comment]
i think this is due the weak dependency on the 2 bootstrap packages
(official one + nemo64 version)  -- i'm not quite sure how to resolve this
except by making the dependency strong on one package, but that would mean
we need 2 versions of this package (one for each bootstrap package)

**Comment by KaleenAli** [comment]
Thank you so much for responding so quickly, I really appreciate it!  

So I went back and looked at why I have nemo64:bootstrap and turns out I don't think I need it.  Have you had any experience totally get rid of a package (are there any other steps besides deleting from the packages file (above)?)? 

I'm semi-new to meteor and don't mind just using the 'official' bootstrap that your package relies on, I'm just not quite sure how to cut out the second bootstrap package all together.  

In my first attempt, I removed the nemo64:bootstrap package and re-ran meteor.  The same error I wrote about originally is still happening, but I figure that could be because I'm not removing everything I need to be??
